### ISSUE TITLE
BCA model assigns Preserve treatment even when rut is above 20mm

### ISSUE THREAD

**Issue by fritzjoostenz** [fritzjoostenz/cassandra_default_road_model#1]
Need to refine the BCA objective function so that the objective is penalised when the rut goes above a certain threshold. There seems to be something like this in place already, but the penalty should ramp up more radically when rut goes above say 12 mm (definitely 15mm!!) so that in such cases the objective function will become such that Preserve treatments (for Seals at least) are effectively cancelled.

**Comment by Potholer15** [comment]
Can we put a pin on this one. I will have a play and trial and we will discuss at later date (semi regular tech/development forums?) The ethos of the BCA is to let the economics decide and I'm a bit reluctant to have add-on factors to sway economic choices. I am thinking maybe a 'holding' stream, rather than 'preservation', for such cases. Holding generally comes into play when programme is under budget duress. And I would like to see more condition factors like this in the Holding outcomes for BCA. I don't believe the issue raised is present in the MCDA type analysis.

**Comment by fritzjoostenz** [comment]
Thanks Dean - replying here to see whether it comes into GitHub.

100% OK to just keep an eye on this. Bear in mind the data being used is largely generated/mythical, so it may just be an edge case.
________________________________

**Comment by Potholer15** [comment]
That's right. I would be quite OK if rutting is high but every other condition indicator is all well and good. I think two or more distresses need to be present before condemning a section, context is everything. I will keep this issue pinned. As mentioned, I would happier if this example of a poor condition flag (if backed by another flag) translated into holding reseal rather than preservation reseal.

**Comment by fritzjoostenz** [comment]
Thanks Dean - agreed any happy for this to be pinned for now. Will keep the issue open though.
________________________________
